recipes/showit/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set SHOWIT_VERSION = "1.1.4" %}
+
+package:
+  name: showit
+  version: {{ SHOWIT_VERSION }}
+
+source:
+  url:  https://github.com/freeman-lab/showit/archive/ef76425797c71fbe3795b4302c49ab5be6b0bacb.zip
+  sha1: 64484b77fba4fb641daa964ff28b6cb151810222
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - setuptools
+    - python
+  run:
+    - python
+    - matplotlib
+    - numpy !=1.13.0
```

Comment:
last question, this numpy version is not allowed but any later one is ok?

Reply:
This is from their requirements.txt file, due to the version in travis leading to:

> TypeError: numpy boolean subtract, the `-` operator, is deprecated, use the bitwise_xor, the `^` operator, or the logical_xor function instead.

See https://github.com/spacetx/starfish/pull/17

Reply:
I read it like only <1.13 is supported as the operator is deprecated :(

Reply:
We only intended to disallow that particular version.

Reply:
@ambrosejcarr numpy 1.13.1 does support the operator again?